### PR TITLE
aws release 19.3.2

### PR DESCRIPTION
**Pull request by whites11**
### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

**Comment by whites11** [comment]
apart from release notes, :green_circle: for releasing

**Review by LolloneS — APPROVED** [review]

File: aws/v19.3.2/release.yaml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    version: 0.0.1
+  - catalog: giantswarm
+    dependsOn:
+    - aws-cloud-controller-manager
+    - cilium
+    - coredns
+    - prometheus-operator-crd
+    - vertical-pod-autoscaler-crd
+    name: security-bundle
+    version: 1.4.2
```
Comment (fhielpos) [inline aws/v19.3.2/release.yaml:154]:
```suggestion
    version: 1.6.2
```

Can we bump this?

**Comment by fhielpos** [comment]
Sorry guys I merged this by accident 🤦 

https://github.com/giantswarm/releases/pull/1213

**Pull request merged**